operations/research-and-development/product/product-technical-writing-team-handbook.md

```diff
@@ -28,6 +28,16 @@ The Technical Writing team focuses primarily on the User Guide, Admin Guide, and
   * Community Help Wanted doc issues
   * Documentation Handbook and Style Guide maintenance
 
+#### @sebastian.faase:
+
+- Identifying and ranking gaps and pain points among customers
+- Establish difference in voice and tone between CE and technical documents
+- Create customer centered guides for following critical points in deployment
+  - Initial setup
+  - License use after trial
+  - License renewal
+  - Reducing customers need for MM Support
```

Comment:
```suggestion
  - Reducing customer need for Mattermost support
```
Are we saying we're decreasing it overall or in the context of CE? Is there a specific type of need we're reducing?